mscore/toolbarEditor.cpp

```diff
@@ -18,9 +18,9 @@
 namespace Ms {
 
 static const char* toolbars[] = {
-      QT_TRANSLATE_NOOP("toolbar", "Note Input"),
-      QT_TRANSLATE_NOOP("toolbar", "File Operations"),
-      QT_TRANSLATE_NOOP("toolbar", "Playback Controls")
+      "Note Input",
+      "File Operations",
+      "Playback Controls"
```

Comment:
Isn't this rather untranslating these strings?

Reply:
Yes, I'm untranslating them here, because they are already translated in line 50. So they were actually translated twice.

Reply:
They are not translated **here**, just extracet by `lupdate`. The translation happens at `qApp->translate("toolbar",...)`

Reply:
`QT_TRANSLATE_NOOP()` is just for `lupdate` and on compile (or rather on pre-process) just evaluates to the 2nd arg.

Reply:
`qApp->translated(context, string)` then pick up `string` in ist translated form.

Reply:
Ah, I see, I will have another look at it!